### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Arkadyus**
Cleaned up the imports
Added better refreshing of the calendar
Added easier event editing

This work is not linked to a PR, it is work I did to make the demo look cleaner and it definitely makes sense to add it on main

### CODE REVIEW

File: app/src/main/java/com/github/houseorganizer/houseorganizer/EventsAdapter.java [hunk 1]
```diff
@@ -111,26 +112,43 @@ private void eventButtonListener(Event event, View v) {
...
+                            .setPositiveButton(R.string.confirm, (editForm, editFormId) -> editEventAndDismiss(event, editForm, dialogView, position))
                             .setNegativeButton(R.string.cancel, (editForm, editFormId) -> dialog.dismiss())
                             .show();
                 }).show();
     }
 
-    private void editEventAndDismiss(String eventId, DialogInterface editForm, View dialogView, FirebaseFirestore db) {
+    private View createEditDialog(View v, Event event) {
+        LayoutInflater inflater = LayoutInflater.from(v.getContext());
+        View retView = inflater.inflate(R.layout.event_creation, null);
```
Comment (sansive) [inline app/src/main/java/com/github/houseorganizer/houseorganizer/EventsAdapter.java:134]: There is still a warning here about passing "null" as the view root

**Review by sansive — COMMENTED** [review]
There are still some warnings in xml files too

**Review by Geno22 — APPROVED** [review]
Thanks the project really needed that !

File: app/src/main/java/com/github/houseorganizer/houseorganizer/EventsAdapter.java (hunk 1 above)
Comment (Arkadyus) [reply app/src/main/java/com/github/houseorganizer/houseorganizer/EventsAdapter.java:156]: Fixing this in next commit

**Review by sansive — APPROVED** [review]
Nice work!

**Pull request merged**